src/card-list.js

```diff
@@ -11,8 +21,8 @@ export default function CardList(props) {
       {props.cards.length > 0 &&
       <div className="container-fluid">
         <div className="row justify-content-start">
-          {props.cards.map((card, index) => {
-            return <Card key={index} card={card}/>
+          {props.cards.map((card) => {
+            return <Card id={card.cardId} function={cardInfo} key={card.cardId} card={card}/>
```

Comment:
Do not name things `array`, `object`, or `function`. Use your problem domain (a study app) for identifier ideas.